spec/lib/dttp/parsers/contact_spec.rb

```diff
@@ -5,7 +5,7 @@
 module Dttp
   module Parsers
     describe Contact do
-      describe ".to_user_attributes" do
+      describe "#to_user_attributes" do
```

Comment:
To be pedantic, `.` is the right choice over `#`, since this is a class method.
https://www.betterspecs.org/#describe

Reply:
of course! thank you, changed 🙏